src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -115,8 +117,10 @@ public void save(AddressBook addressBook) throws StorageOperationException {
      * Loads data from this storage file.
      *
      * @throws StorageOperationException if there were errors reading and/or converting data from file.
+     * @throws FileNotFoundException if file is not found during runtime
+     * 
      */
-    public AddressBook load() throws StorageOperationException {
+    public AddressBook load() throws StorageOperationException, FileNotFoundException{
```

Comment:
Check for exception at the right location; while saving/storing and not loading
The purpose is to check accidental file deletes and not check missing file initially.